contrib/langchain/pyproject.toml

```diff
@@ -12,7 +12,7 @@ license = "MIT"
 requires-python = ">=3.10"
 dependencies = [
     "arcadepy>=1.7.0",
-    "langchain-core>=0.3.49,<0.4",
+    "langchain-core>=1.0.0",
```

Comment:
If you could tie these to the major version, then it will help prevent a dependency's breaking change from breaking this langchain-arcade library.